frontend/e-buddy/src/app/(demo)/bots/setup/page.tsx

```
@@ -0,0 +1,159 @@
+                </button>
+              </div>
+            </div>
+          )}
+
+          {/* Step 2: URL Setup */}
+          {step === 2 && (
+            <div>
+              <h2 className="text-2xl font-semibold text-gray-800 mb-4">Step 2: URL for Scrape</h2>
+              <form className="space-y-6">
+                <div>
+                  <label className="block text-gray-700 mb-2">URL for Scrape</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)